controllers/novels.js

```diff
@@ -0,0 +1,32 @@
+const models = require('../models')
+
+const getAllNovels = async (req, res) => {
+  try {
+    const novels = await models.Novels.findAll({
+    })
```

Comment:
This line should have been in you getAllNovels to satisfy the last call(GET) on the README

 include: [{ model: models.Authors }, { model: models.Genres }]